### DISCUSSION TITLE
Long descriptions for accessible images

### DISCUSSION THREAD

**Discussion by lizneely** [thegetty/quire#655] [Get Help]
Hello!

I am working with Scribely to get text alternatives written to make images accessible in a publication. They have recommended long descriptions for a set of images as required when more details are necessary to provide an accessible "text alternative" to the image. Quire has a great place for alt text, but is anyone using one of the techniques specifically for long description as outlined in guidance from the W3C: https://www.w3.org/WAI/tutorials/images/complex/?

If so, I'd love to hear! I can put these in alt text, but it seems like that is a short-term solution.

Thank you,
Liz

**Comment by salger1** [comment]
I don't have a answer for you, but I would definitely be interested in hearing if any solutions came up. This is something we will potentially be grappling with in one of our catalogues.
